fsw/crypto_sadb/sadb_mariadb_admin_scripts/create_sadb.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE IF NOT EXISTS sadb;
```

Comment:
Like the directory structure for the SADB / maria / SQL scripts.  We were just talking this week about doing something like this to keep things separated for the different SA implementations.